src/displayBackend/wayland/Display.cpp

```diff
@@ -408,8 +408,14 @@ void Display::sWaylandLog(const char* fmt, va_list arg)
 void Display::sRegistryHandler(void *data, wl_registry *registry, uint32_t id,
 							   const char *interface, uint32_t version)
 {
-	static_cast<Display*>(data)->registryHandler(registry, id, interface,
-												 version);
+	try
+	{
+		static_cast<Display*>(data)->registryHandler(registry, id, interface, version);
+	}
+	catch(const std::exception& err)
+	{
+		LOG("Wayland", ERROR) << err.what();
```

Comment:
Are you sure that this is a right way to handle the exception?
Looks like you just ignoring it.

I think, that we should properly handle it. Or stick to strategy "fail fast" and crash the program.